frontend/src/components/VHeader/VHeaderMobile/VSearchBarButton.vue

```diff
@@ -66,6 +67,13 @@ export default defineComponent({
       type: Boolean,
       default: false,
     },
+    /**
+     * The label to use as accessible name for the button (aria-label).
+     */
+    label: {
+      type: String,
```

Comment:
If we make the `label` prop accept translate results, we can drop the `.toString` from all the different places where it is passed.

```typescript
import { TranslateResult } from 'vue-i18n'
```

```suggestion
      type: [String, Object] as PropType<string | TranslateResult>,
```